source/MRVoxels/MRObjectVoxels.cpp

```diff
@@ -34,7 +34,7 @@ void ObjectVoxels::construct( const SimpleVolume& simpleVolume, const std::optio
     mesh_.reset();
     activeVoxels_.reset();
     activeBounds_.reset();
-    vdbVolume_.data = simpleVolumeToDenseGrid( simpleVolume, cb );
```

Comment:
minmax is calculated in 
https://github.com/MeshInspector/MeshLib/pull/3595/files#diff-ebe3184c010cdbecab4fd34c6a7d92c96e44ac42af9e96fa7ecc1990543dd909R41-R46

so better use correct value instead of 0.0 (also please note that if normalPlusGrad, background should be `max`)